src/commands/init.js

```diff
@@ -155,6 +155,7 @@ const loadAbiFromEtherscan = async (network, address) =>
   await withSpinner(
     `Fetching ABI from Etherscan`,
     `Failed to fetch ABI from Etherscan`,
+    `Warnings fetching ABI from Etherscan`,
```

Comment:
To maintain consistency with the warnings text phrasing should we use "while" here? Making it: `Warnings while fetching ABI from Etherscan`.